runtime/oti/j9nonbuilder.h

```diff
@@ -2125,6 +2125,13 @@ typedef struct J9ConstantPool {
 #define J9CPTYPE_METHODHANDLE  14
 #define J9CPTYPE_ANNOTATION_UTF8  15
 
+/* 
+ * INTERFACE_STATIC and INTERFACE_INSTANT cpType are used to track the classfile tag of a methodref [CFR_CONSTANT_InterfaceMethodref] 
+ * This will be used to check for cp consistency during resolve time
```

Comment:
`We need them to support the correct initializer for the constantpool due to the way we pre-initialize the cp entries`

Can you add a comment like ^^^^ to the comment here?  It makes it clear why we don't just use `J9CPTYPE_INTERFACE`